html/feed/feed.js

```diff
@@ -386,7 +386,21 @@ function generateProfileCards(data, container) {
     container.appendChild(card);
   });
 
-  
+
   generatePage();
 }
 
+const select =document.getElementById ("likedPost");
+const likedPost = document.querySelector ("#likedPost");
+select.addEventListener ('click',async () => {
```

Comment:
is this a button or a  select with options in it? if you use click on that element everything you click it the function will run.
would reccomend reading up on eventlisteners : ) in this case maybe 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event